src/bundle/Resources/public/js/scripts/helpers/browser.helper.js

```diff
@@ -0,0 +1,7 @@
+const { userAgent } = window.navigator;
+const isChrome = () => userAgent.indexOf('Chrome') != -1;
```

Comment:
Shouldn't you check here if it's not safari?
as in Safari you check both, so I guess here you should as well

Reply:
No, chrome always have safari but safari not always have chrome.